### ISSUE TITLE
Close button broken in Browse after multiple Slide Lefts

### ISSUE THREAD

**Issue by Slummi** [strongbox-password-safe/Strongbox#184]
Hi,

I tested the new pinning feature.
It works very well, but how can I unpin a pinned entry? I can‘t find any option to do this (e.g. after swiping left).

The only way I found is to delete the entry and restore it from recycle bin, which is a little bit circumstantial.

I think there is another option, but I can‘t find it.

**Comment by Slummi** [comment]
That‘s interesting. I have no unpin button in neither case.
Maybe it‘s an issue with the German localization and the button is only missing in the localized version?!

**Comment by Slummi** [comment]
Very strange... After I had rotated to landscape mode the unpin button was there. After returning to portrait mode it‘s still there. Restarted the app and it‘s still there. I don‘t know what was wrong. Crazy... :-/

**Comment by Slummi** [comment]
Haha, that bug is as crazy as another one I had yesterday.
After returning from editing mode (multi selection in the entries view) the button in the top left corner was still labeled with „cancel“ instead of „close“ and had no function. So I had to kill the app to close the database. But I can‘t reproduce this.

Yes, it does fit (see screenshots). I tested it on iPhone X and iPhone 7.
Another question: is it intended that you can‘t pin entries in a read-only databse? There‘s no option to swipe left.

![660CBA72-C0DB-49C2-B1E0-16929F8ADE8E](https://user-images.githubusercontent.com/42265381/63777403-57f02c80-c8e3-11e9-93c2-96a50699e222.png)
![546FEC04-01FE-4320-96AF-5A5675E37DA9](https://user-images.githubusercontent.com/42265381/63777404-5888c300-c8e3-11e9-816b-6604e8b8fd07.png)

**Comment by Slummi** [comment]
Ha, I solved the cancel/close puzzle. :-)
Found two ways to reproduce that bug:

1. Swipe left on any entry so that the actions are shown. Then swipe left on any other entry and the cancel button is there for ever.

2. Swipe left on any entry and then quickly swipe left on another one while the actions from the first entry are sliding back and the cancel button stays also there.

Seems to be a kind of race condition. Maybe there are some more ways to that bug.
